src/agents/common.jl

```diff
@@ -192,6 +190,8 @@ function HEMData(input_filename::String; epsilon::AbstractFloat = 1.0E-3)
         description = "Time"
     )
     # TODO: Remove hard-coding. (This start year is also specified in HEMData.jl)
+    # Perhaps requires loading the HEMData config, which currently isn't stored in
+    # the input directory.
```

Comment:
Created issue https://github.com/nrel-hem/HolisticElectricityModelData.jl/issues/28 to provide the needed data and issue https://github.com/nrel-hem/HolisticElectricityModel.jl/issues/50 to use the data when they are available.